app/javascript/packages/components/troubleshooting-options.jsx

```diff
@@ -11,17 +11,20 @@ import { BlockLink } from '@18f/identity-components';
 /**
  * @typedef TroubleshootingOptionsProps
  *
- * @prop {string} heading
```

Comment:
Should we bring this option (back into) alignment b/w JSX and ERB?

Reply:
fixed in `6516040`